cfme/tests/webui/test_general_ui.py

```diff
@@ -288,6 +289,47 @@ def test_add_button_on_provider_all_page(
     assert displayed_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1475553])
+def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
```

Comment:
**Question** Why pull a temp_appliance for this case? Is this test destructive to an appliance?

Reply:
I need an appliance where SMTP has never been configured, I am using temp_appliance to make sure I get one.